w3lib/url.py

```diff
@@ -34,9 +34,12 @@ def _quote_byte(error):
 
 _safe_chars = RFC3986_RESERVED + RFC3986_UNRESERVED + EXTRA_SAFE_CHARS + b'%'
 
+_ascii_tab_newline_re = re.compile(r'[\t\n\r]')  # see https://infra.spec.whatwg.org/#ascii-tab-or-newline
```

Comment:
Since we are linking to the ‘living standard’, which changes constantly, I think we should not link directly to it, but through the [Wayback Machine](https://web.archive.org/). Both here and on the docstring below.

Reply:
Wayback Machine is being blocked and unblocked in Russia, so I'm fine with whatwg link :)